src/test/java/com/google/sps/data/PollingStationTest.java

```diff
@@ -57,6 +57,174 @@ public void testFromJSONObject() throws Exception {
     Assert.assertEquals(pollingStation.getLocationType(), "pollingLocation");
   }
 
+  @Test
+  public void line1FieldMissing_testFromJsonObject() {
+    JSONObject pollingStationJsonObject =
+        new JSONObject(
+            "{\"id\": \"pollingId\","
+                + "\"address\": {\"locationName\": \"name\","
+                + "\"line2\": \"2\",\"line3\": \"3\",\"city\": \"city\",\"state\": \"state\","
+                + "\"zip\": \"zip\"},\"pollingHours\": \"-\",\"name\": \"pollingStation\","
+                + "\"startDate\": \"start\",\"endDate\": \"end\"}");
+
+    PollingStation pollingStation =
+        PollingStation.fromJSONObject(pollingStationJsonObject, "pollingLocation");
+
+    Assert.assertEquals(pollingStation.getName(), "pollingStation");
+    Assert.assertEquals(pollingStation.getAddress(), "2 3, city, state zip");
+    Assert.assertEquals(pollingStation.getPollingHours(), "-");
```

Comment:
What does data look like for actual polling hours? Can we use an example that's similar to the hours that civic api might return?

Reply:
"-" is something that is returned by the civic information API -- it's actually what I get when I input my address.